controllers/session.go

```diff
@@ -15,31 +15,141 @@
 package controllers
 
 import (
-	"fmt"
+	"bytes"
+	"encoding/json"
+	"github.com/astaxie/beego"
+	"io"
+	"io/ioutil"
+	"mime/multipart"
+	"net/http"
+	"strconv"
 
 	"github.com/casdoor/casdoor-go-sdk/auth"
 )
 
-var sessionMap = map[string]int64{}
+type Session struct {
+	Owner       string `xorm:"varchar(100) notnull pk" json:"owner"`
+	Name        string `xorm:"varchar(100) notnull pk" json:"name"`
+	Application string `xorm:"varchar(100) notnull pk default ''" json:"application"`
+	CreatedTime string `xorm:"varchar(100)" json:"createdTime"`
+
+	SessionId []string `json:"sessionId"`
+}
+
+func addUserSession(claims *auth.Claims, sessionId string) {
```

Comment:
Put all code to: https://github.com/casdoor/casdoor-go-sdk , re-use the existing code base